src/cpp/ext/filters/census/grpc_plugin.cc

```diff
@@ -51,13 +54,6 @@ void RegisterOpenCensusPlugin() {
                   &grpc::internal::OpenCensusClientFilter::kFilter);
               return true;
             });
-        builder->channel_init()->RegisterStage(
```

Comment:
Why do we no longer need the server side to be a filter but we still need the client side to be a filter?

Reply:
As discussed in the design review, the method of creating call tracers are going to be different on the client-side and the server-side.
On the client-side, we are going to have an interceptor inject this. (I know it's a filter right now, but it'll move to an interceptor). On the server-side, we will have a factory that gets invoked by the surface call.